DigitalLearningSolutions.Web/Views/SuperAdmin/AdminAccounts/Index.cshtml

```diff
@@ -0,0 +1,123 @@
+                                                    required="true"
+                                                    hint-text=""
+                                                    css-class="nhsuk-grid-column-full nhsuk-u-width-full"
+                                                    default-option=""
+                                                    select-list-options="@ViewBag.UserStatus" />
+                                </div>
+                                <div class="nhsuk-grid-column-one-quarter">
+                                    <vc:select-list asp-for="@nameof(Model.Role)"
+                                                    label="Role"
+                                                    value="@Model.Role"
+                                                    required="true"
+                                                    hint-text=""
+                                                    css-class="nhsuk-grid-column-full nhsuk-u-width-full"
+                                                    default-option=""
+                                                    select-list-options="@ViewBag.Roles" />
+                                </div>
+                                <div class="nhsuk-grid-column-one-half">
+                                    <vc:select-list asp-for="@nameof(Model.CentreID)"
+                                                    label="Centre"
+                                                    value="@Model.CentreID.ToString()"
+                                                    hint-text=""
+                                                    required="true"
+                                                    css-class="nhsuk-grid-column-full  nhsuk-u-width-full"
+                                                    default-option=""
+                                                    select-list-options="@ViewBag.Centres" />
+                                </div>
+                            </div>
+                        </div>
+                    </form>
+                    @if (Model.NoDataFound)
+                    {
+                        <p class="nhsuk-u-margin-top-4" role="alert">
+                            <b>No admin found.</b>
+                        </p>
+                    }
+                    else
+                    {
+                        <partial name="SearchablePage/_TopPagination" model="Model" />
+                        <partial name="SearchablePage/_ResultCountAndPageAlerts" model="Model" />
+                        <div id="searchable-elements">
+                            @foreach (var admin in Model.Admins)
+                            {
+                                <partial name="_SearchableAdminAccountsCard" model="admin" />
+                            }
+                        </div>
+                        <partial name="SearchablePage/_BottomPagination" model="Model" />
```

Comment:
When Next page link is clicked, the applied filters are removed. Chhose a filter item -> click Search -> click Next : Removes filter and moves to page 2

Reply:
filters are maintained on the next button click.